echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/controller/NotificationController.groovy

```diff
@@ -32,8 +38,25 @@ class NotificationController {
   @Autowired(required=false)
   Collection<NotificationService> notificationServices
 
+  @Autowired
+  InteractiveNotificationCallbackHandler callbackHandler
+
   @RequestMapping(method = RequestMethod.POST)
   EchoResponse create(@RequestBody Notification notification) {
     notificationServices?.find { it.supportsType(notification.notificationType) }?.handle(notification)
   }
+
+  @RequestMapping(
+    value = '/callbacks/{source}',
+    method = RequestMethod.POST,
+    consumes = MediaType.APPLICATION_FORM_URLENCODED_VALUE,
+    produces = MediaType.APPLICATION_JSON_VALUE
+  )
+  // Required due to https://github.com/spring-projects/spring-framework/issues/22734
```

Comment:
Never mind. I initially was trying to reuse `WebhooksController`.

Reply:
So is this code going away?

Reply:
No, just the comment. 😃